Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -94,7 +96,7 @@ final class StatusTableViewController: LoopChartsTableViewController {
             },
         ]
 
-        deviceManager.$isClosedLoop
+        closedLoopStatusObservable.$isClosedLoop
```

Comment:
Since `isClosedLoop` is a single value, what is the benefit in wrapping it in an object (`closedLoopStatusObservable`)?  Seems like we have a new type, another accessor level to get to the publisher, and more code. I'm familiar with using ObservableObject in the SwiftUI context, but it feels odd for normal state management. Maybe I'm missing some benefit?

Reply:
You are right that `ObservableObject` isn't needed here. 

As for encapsulating in an object, isn't this needed to ensure all reference that same published property? Likely I have a misunderstanding, but I'm not sure how else this can be accomplished.